### DISCUSSION TITLE
Question: How does one configure the internal padding of a button without grid()?

### DISCUSSION THREAD

**Discussion by 32-git** [TomSchimansky/CustomTkinter#365] [General]
Currently, I find myself in a dilemma, which is the following:

I have a CTk button which I would like to add internal padding to. Based on what I know so far, I would need to use grid() to make that possible. However, I have used the 'relx' and 'rely' attributes of place() to position my button and would like to keep it that way.

Hence, my question: Is there another way of adding internal padding to a button? (preferably one where place() is still usable)

**Comment by TomSchimansky** [comment]
Currently there is not, the internal padding of the button is always the corner_radius or if if the corner_radius is smaller than 2, its 2.

**Reply by 32-git** [reply]
Thank you for the prompt response. 

So you mean to say, that the corner_radius also controls the padding. Therefore, a higher corner_radius, would add more internal padding. After some testing, this does seem to be the case, however only horizontally.

Is there any way to do the same vertically?

**Comment by TomSchimansky** [comment]
Yes

32-git ***@***.***> schrieb am So., 7. Aug. 2022 16:11:

**Reply by 32-git** [reply]
I must apologise for the constant questions. But how would I do this?
